aten/src/ATen/native/BatchLinearAlgebra.h

```diff
@@ -6,7 +6,11 @@
 #include <ATen/native/TransposeType.h>
 
 
-namespace at { namespace native {
+namespace at {
+
+struct TensorIterator;
```

Comment:
Why is this needed?

Reply:
I was getting a compilation error saying the symbol was not found, so I had to forward declarate it. Now, I do not know what caused it from this PR, as it seems unrelated.

That being said, TI was not being imported in this file, and it should be there, as `lu_unpack` has TI in its signature.

Reply:
In fact, I was missing this in the previous PR, where I moved `lu_unpack` to this file.